src/navigation/Navigator.tsx

```diff
@@ -59,6 +60,8 @@ const Navigator = (): JSX.Element => {
   const [isPressed, setIsPressed] = React.useState<boolean>(false)
   const [isHomeButtonPressed, setIsHomeButtonPressed] = React.useState<boolean>(false)
 
+  initSentry()
```

Comment:
I think a better place for this would be App.tsx. In Integreat we just have this in the navigator because it was customizable in the intro slides whether ti should/should not be enabled.